packages/rum-react/README.md

```diff
@@ -4,3 +4,113 @@
 > This package is experimental. Use at your own risk.
 
 This package provides React and React ecosystem integrations for Datadog Browser RUM.
+
+## Installation
+
+```bash
+npm install @datadog/browser-rum @datadog/browser-rum-react
+```
+
+## Usage
+
+### Initialization
+
+To enable the React integration, pass the `reactPlugin` to the `betaPlugins` option of the `datadogRum.init` method:
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum'
+import { reactPlugin } from '@datadog/browser-rum-react'
+
+datadogRum.init({
+  applicationId: ...,
+  clientToken: ...,
+  ...
+  betaPlugins: [reactPlugin()],
+})
+```
+
+### Error Tracking
+
+To track React component rendering errors, use one of the following:
+
+- A basic `ErrorBoundary` component (see [React documentation][1]) that catches errors and reports them to Datadog.
+- A function that you can use to report errors from your own `ErrorBoundary` component.
+
+#### `ErrorBoundary` usage
+
+```javascript
+import { ErrorBoundary } from '@datadog/browser-rum-react'
+
+function App() {
+  return (
+    <ErrorBoundary fallback={ErrorFallback}>
+      <MyComponent />
+    </ErrorBoundary>
+  )
+}
+
+function ErrorFallback({ resetError, error }: { resetError: () => void; error: unknown }) {
+  return (
+    <p>
+      Oops, something went wrong! <strong>{String(error)}</strong> <button onClick={resetError}>Retry</button>
+    </p>
+  )
+}
+```
+
+#### Reporting React errors from your own `ErrorBoundary`
+
+```javascript
+import { addReactError } from '@datadog/browser-rum-react'
+
+class MyErrorBoundary extends React.Component {
+  componentDidCatch(error: Error, errorInfo: React.ErrorInfo) {
+    addReactError(error, errorInfo)
+  }
+
+  render() {
+    ...
+  }
+}
+
+```
+
+### React Router integration
+
+`react-router` v6 allows you to declare routes using the following methods:
+
+- Create routers with [`createMemoryRouter`][2], [`createHashRouter`][3], and [`createBrowserRouter`][4] functions.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
- Create routers with [`createMemoryRouter`][2], [`createHashRouter`][3], or [`createBrowserRouter`][4] functions.
```